repos/spack_repo/builtin/packages/casacore/package.py

```diff
@@ -51,6 +51,7 @@ class Casacore(CMakePackage):
     variant("shared", default=True, description="Build shared libraries")
     variant("tablelocking", default=True, description="Enable table locking")
     variant("threads", default=True, description="Use mutex thread synchronization")
+    variant("tests", default=False, description="Build and run tests")
```

Comment:
The standard Spack way to build and run unit tests is with `spack install --test=root`. Seeing that setting `+tests` doesn't actually run or install the tests, I think that it's not a legitimate variant, and we should stick with keying off the `--test=root` argument. I think that's `self.run_tests`.